inflight.go

```diff
@@ -171,18 +175,9 @@ func callAndCacheBatch[V, T any](ctx context.Context, c *Client[T], opts callBat
 	response := make(map[string]V, len(opts.ids))
 	for call, callIDs := range callIDs {
 		call.Wait()
-		// It could be only cached records here, if we we're able
-		// to get some of the IDs from the distributed storage.
-		if call.err != nil && !errors.Is(call.err, ErrOnlyCachedRecords) {
-			return response, call.err
-		}
-
-		if errors.Is(call.err, ErrOnlyCachedRecords) {
-			err = ErrOnlyCachedRecords
-		}
 
-		// We need to iterate through the values that we want from this call. The
-		// batch could contain a hundred IDs, but we might only want a few of them.
+		// We need to iterate through the values that WE want from this call. The batch
+		// could contain hundrdreds of IDs, but we might only want a few of them.
```

Comment:
Typos: `we`, `hundreds`

Reply:
Ah, thank you for spotting this! The `WE` was intentional but _ hundrdreds_ was not. I'll update!